e2e_tests/src/raw_request.rs

```diff
@@ -13,7 +13,7 @@ const MAX_BODY_SIZE: usize = 1 << 31;
 #[derive(Copy, Clone, Debug)]
 pub struct RawRequestClient;
 
-static SOCKET_PATH: &str = "/tmp/parsec/parsec.sock";
+static SOCKET_PATH: &str = "/run/parsec/parsec.sock";
```

Comment:
I think it might be worth to have `test` feature that put the `SOCKET_PATH` under `/tmp/parsec.sock` (with no folder) that way:
* developpers can easily just run the Parsec service without having to create a folder
* if you already have a Parsec service installed you can still develop with that path
* environments where `/run` is not available can still test Parsec

Also this new feature should maybe be merged with the existing `no-parsec-user-and-clients-group` to group all features used with testing?

Reply:
And similar changes would be needed on the client, and same thing with the `no-fs-permission-check` feature